annlite/executor.py

```diff
@@ -114,7 +114,7 @@ def __init__(
 
     @requests(on='/index')
     def index(
-        self, docs: Optional[DocumentArray] = None, parameters: dict = {}, **kwargs
+        self, docs: Optional[DocumentArray] = None, parameters: dict = None, **kwargs
```

Comment:
In this way, we should use `parameters: Optional[dict] = None`

Reply:
Basically, I agree with this change. But I can not completely see the advances of it. Could you give us some more information about this change?